### PR TITLE
Update Instructions on Contributing

### PR DESCRIPTION
**Pull request by marrero984**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,6 +7,43 @@ The _lmscontent directory is connected to Distelli; commits will be automaticall
...
+Clone this repo to your local development environment.
+
+    git clone https://github.com/puppetlabs/courseware-lms-content.git
+    
+Each directory in _lmscontent represents a learning component in LearnDot.  The content.md, description.md, and summary.md are converted to HTML and uploaded to the appropriate field; the remaining LearnDot fields are populated via the metadata.json file.  **Do not edit the metadata.json file,** unless you are familiar with LearnDot's idiosyncrasies.
+
+### Updating Existing Materials to Staging
+
+* Using the editor of your choice, update the relevant markdown files for a learning component.
+* Once you are done editting, stage the files for commit.
```
Comment (carthik) [inline README.md]: editing

File: README.md [hunk 2]
```diff
@@ -7,6 +7,43 @@ The _lmscontent directory is connected to Distelli; commits will be automaticall
...
+  * This assumes that you have not forked this repo and updated your remote names.
+
+You may now log into the [staging instance](https://puppetlabs-staging.trainingrocket.com/) and view your changes.
+
+### Creating a New Learning Component
+
+* Create a new directory within _lmscontent.
+  * The directory name is not too important, but it is suggested that it match the URL endpoint where the component will be listed.
+* Copy the contents of _lmscontent/example into the new directory.
+* Edit the following fields in the metadata.json:
```
Comment (greglarkin) [inline README.md]: Does the `urlName` field also need to be updated? I thought that was mentioned a while back, too. @marrero984

File: README.md [hunk 3]
```diff
@@ -7,6 +7,43 @@ The _lmscontent directory is connected to Distelli; commits will be automaticall
 
 This repo only contains markdown, images, and transcripts; for video files, please see TBD.
 
+Getting Started
+=======================
+
+Based on our current workflow, you do not need to fork this repo.  You may fork it if you want to and submit PRs through the standard git workflow, but it is **not** required.
```
Comment (greglarkin) [inline README.md]: Zack would know for sure, but I don't think merge commits from a PR will trigger the Distelli build automatically, although I think he may have that on his list to fix. @marrero984

File: README.md (hunk 2 above)
Comment (marrero984) [reply README.md]: The `urlName` should be left blank on creation; LearnDot will auto populate based on the `name` field

File: README.md (hunk 3 above)
Comment (marrero984) [reply README.md]: @acidprime Do commits merged through a PR trigger the pipeline?

Comment (marrero984) [reply README.md]: @greglarkin This has been confirmed.  Merging a PR does trigger the pipeline deployment

**Pull request merged**